docs/release/trg-0/trg-1-09.md

```diff
@@ -0,0 +1,42 @@
+---
+title: TRG 1.09 - UX consistency / Styleguide for UI
+sidebar_position: 1
+---
+
+:::caution
+Proposed release date: 15th of January 2024
+:::
+
+| Status | Created     | Post-History                                          |
+|--------|-------------|-------------------------------------------------------|
+| Draft  | 10-Jan-2024 | Transferred from Release Acceptance Criteria into TRG |
+
+## Why
+
+Ensuring UX consistency through a style guideline is essential for a unified and user-friendly experience across front end modules.
+
+## Description
+
+- User Interfaces **should** align with the Tractus-X Style Guidelines
+
+> User interface style review has been executed.
```

Comment:
This quote block reads really bad when rendered (as all line breaks will be ignored. Transform into a list instead?

Reply:
Yep, Julia wants to change it after the merge. Right now with Kelajas original PR this would not have been possible.